src/main/java/org/sunbird/workflow/service/impl/OrganisationWorkFlowServiceImpl.java

```diff
@@ -1,24 +1,73 @@
 package org.sunbird.workflow.service.impl;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
+import org.sunbird.workflow.config.Configuration;
 import org.sunbird.workflow.config.Constants;
 import org.sunbird.workflow.models.Response;
 import org.sunbird.workflow.models.SearchCriteria;
 import org.sunbird.workflow.models.WfRequest;
 import org.sunbird.workflow.service.OrganisationWorkFlowService;
 import org.sunbird.workflow.service.Workflowservice;
 
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
 @Service
 public class OrganisationWorkFlowServiceImpl implements OrganisationWorkFlowService {
 
     @Autowired
     private Workflowservice workflowService;
 
+    @Autowired
+    private Configuration configuration;
+
+    @Autowired
+    private RequestServiceImpl requestServiceImpl;
+
+    private Logger logger = LoggerFactory.getLogger(OrganisationWorkFlowServiceImpl.class);
+
+    /**
+     * Service method to handle the creation of Users.
+     *
+     * @param rootOrg   - Root Organization Name ex: "igot"
+     * @param org       - Organization name ex: "dopt"
+     * @param wfRequest - WorkFlow request which needs to be processed.
+     * @return - Return the response of success/failure after processing the request.
+     */
     @Override
     public Response createOrgWorkFlow(String rootOrg, String org, WfRequest wfRequest) {
-        Response response = workflowService.workflowTransition(rootOrg, org, wfRequest);
-        return response;
+        Response response = new Response();
+        wfRequest.getUpdateFieldValues().forEach(updateFieldValuesMap -> updateFieldValuesMap.forEach((updateFieldValuesKey, updateFieldValuesValue) -> {
+            if (Constants.EMAIL.equalsIgnoreCase(updateFieldValuesKey) && isUserDetailExists(Constants.EMAIL, updateFieldValuesValue.toString())) {
```

Comment:
updateFieldValuesValue.toString() --> .toString() method should not be called. This creates the chance for NPE.

(String) updateFieldValuesValue --> this is the correct approach.

Reply:
1. Casted the values to String and removed the toString() to avoid NPE.